{"name": "Принять заказ с несуществующим id заказа", "status": "passed", "description": "Попытка принять заказ с несуществующим id заказа должна вернуть ошибку 404", "start": 1701085495183, "stop": 1701085496523, "uuid": "ddf721e2-6fb6-40af-88c9-149003a21b19", "historyId": "bac271c0ad6255fa7bab0a694d9dcb26", "testCaseId": "bac271c0ad6255fa7bab0a694d9dcb26", "fullName": "tests.test_courier.TestCourier#test_accept_order_with_wrong_order_id", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier"}, {"name": "subSuite", "value": "TestCourier"}, {"name": "host", "value": "Ya"}, {"name": "thread", "value": "27764-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier"}]}